src/python/bot/startup/run_bot.py

```diff
@@ -88,7 +116,14 @@ def task_loop():
       update_task.run()
       update_task.track_revision()
 
-      task = tasks.get_task()
+      if environment.get_value('SCHEDULE_UTASK_MAINS'):
+        # If the bot is configured to schedule utask_mains, don't run any other
+        # tasks because scheduling these tasks is more important than executing
+        # any one other task.
```

Comment:
Can you add a TODO to just convert this into a k8s cron? 

Take a look at https://github.com/google/clusterfuzz/blob/master/src/python/bot/startup/run_cron.py and https://github.com/google/clusterfuzz/tree/master/infra/k8s for how this is done. This should be very straightforward and make this a lot cleaner.